dcc-portal-ui/app/scripts/analysis/js/analysisLauncher.js

```diff
@@ -60,13 +66,35 @@
       }
       _this.applyFilter(_this.analysisType);
     };
+    
+    _this.toggleOnco = function(setId, setType) {
+      if (_this.selectedIds.indexOf(setId) >= 0) {
+        _.remove(_this.selectedIds, function(id) {
+          return id === setId;
+        });
+      } else {
+        _this.selectedIds.push(setId);
+      }
```

Comment:
Instead of 

```js
if (_this.selectedIds.indexOf(setId) >= 0) {
  _.remove(_this.selectedIds, function(id) {
    return id === setId;
  });
} else {
  _this.selectedIds.push(setId);
}
```

Try this:

```js
_this.selectedIds = _.xor(_this.selectedIds, [id]);
```